.travis.yml

```diff
@@ -10,14 +10,16 @@ d:
  - dmd-2.068.2
  - ldc-1.0.0-alpha1
  - ldc-0.17.0
-install:
- - if [[ $(dmd --help | head -n 1 | cut -d ' ' -f 4) == 'v2.071.0' && $TRAVIS_OS_NAME == 'linux' ]] ; then export IS_LATEST_PHOBOS=1 ; fi
- - wget -O doveralls "https://github.com/ColdenCullen/doveralls/releases/download/v1.2.0/doveralls_linux_travis"
- - chmod +x doveralls
- - wget -O dscanner "http://releases.dlang.io/dscanner/0.3.0-git/dscanner"
- - chmod +x dscanner
+env:
+ - ARCH="x86_64"
```

Comment:
see below at the matrix includes - `x86_64` is our default architecture and we include a couple of 32bit ones ;-)